models/delivery.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var deliveryModel = require('hmpo-model');
```

Comment:
look at this: https://github.com/UKHomeOffice/passports-model
when you add to the prototype of a constructor, you are essentially defining new methods on any new instance of that constructor/class. So if someone else `new`'s up that constructor/class they'll share any new prototype methods you've added